dist/ExtUtils-ParseXS/lib/perlxstut.pod

```diff
@@ -120,9 +120,9 @@ The file Makefile.PL should look something like this:
     WriteMakefile(
 	NAME         => 'Mytest',
 	VERSION_FROM => 'Mytest.pm', # finds $VERSION
-	LIBS         => [''],   # e.g., '-lm'
-	DEFINE       => '',     # e.g., '-DHAVE_SOMETHING'
-	INC          => '',     # e.g., '-I/usr/include/other'
+	LIBS         => [''], # e.g., '-lm'
+	DEFINE       => '', # e.g., '-DHAVE_SOMETHING'
+	INC          => '-I.', # e.g., '-I. -I/usr/include/other'
```

Comment:
I still do not see why this is being modified.

Reply:
EXAMPLE 1

I take the presumption here that to not include `-I.` would constitute an editing of the `h2xs -A -n Mytest` generated Makefile.PL; which would then perhaps oblige a justification presented to the reader as to the purpose for that editing.

[What has gone on?](https://perldoc.perl.org/perlxstut#What-has-gone-on?)

...In later examples we'll see how we can use h2xs to read header files and generate templates to connect to C routines.

<hr>

[re::engine::Plan9](https://metacpan.org/release/re-engine-Plan9) (Mar 31, 2010) for example includes an .h file in the top level extracted download

Reply:
I see. The explanation for this since it has not been mentioned is that h2xs now generates a dist with ppport.h causing it to include `-I.` in the generated Makefile.PL.